File: src/main/java/net/dontdrinkandroot/gitki/Application.java
package net.dontdrinkandroot.gitki;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.autoconfigure.web.servlet.error.ErrorMvcAutoConfiguration;
import org.springframework.scheduling.annotation.EnableScheduling;

/**
 * @author Philip Washington Sorst <philip@sorst.net>
 */
@SpringBootApplication(exclude = {ErrorMvcAutoConfiguration.class})
@EnableScheduling
public class Application
{
    public static void main(String args[])
    {
        SpringApplication.run(Application.class, args);
    }
}

File: src/main/kotlin/net/dontdrinkandroot/gitki/Application.kt
package net.dontdrinkandroot.gitki

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.autoconfigure.web.servlet.error.ErrorMvcAutoConfiguration
import org.springframework.boot.runApplication
import org.springframework.scheduling.annotation.EnableScheduling

@SpringBootApplication(exclude = [ErrorMvcAutoConfiguration::class])
@EnableScheduling
class Application

fun main(args: Array<String>) {
    runApplication<Application>(*args)
}